src/App.js

```diff
@@ -1,5 +1,36 @@
+import { Console } from '@woowacourse/mission-utils';
+import {
+  DELIMITER_VALIDATOR,
+  CAR_NAME_VALIDATOR,
+  ATTEMPT_TIMES_VALIDATOR,
+} from './inputValidators.js';
+import runRace from './runRace.js';
+import getWinners from './getWinners.js';
+
 class App {
-  async run() {}
+  async run() {
+    const CAR_NAME = await Console.readLineAsync(
+      '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+    );
+
+    DELIMITER_VALIDATOR(CAR_NAME);
+
+    const CAR_NAME_ARRAY = CAR_NAME.trim()
+      .split(',')
+      .map(name => name.trim());
```

Comment:
```suggestion
    const CAR_NAME_ARRAY = CAR_NAME
      .split(',')
      .map(name => name.trim());
```

마지막에 순회하면서 trim메서드를 쓰면 앞에서 trim메서드를 안 써도 되지 않을까요?

Reply:
앗! 제가 착각했네요! 
인풋값의 앞뒤 공백을 먼저 없애고 split을 사용한 후 각 요소에서도 따로 trim을 적용하려 했는데
지금 보니 불필요하게 이중으로 처리한 것 같습니다ㅜㅜ

좋은 피드백 감사합니다! 😃